frontend/public/components/utils/resource-log.tsx

```diff
@@ -243,148 +271,358 @@ export const LogControls: React.FC<LogControlsProps> = ({
+                    id="showFullLog"
+                    data-test="popover-show-full-log"
+                    isChecked={isShowFullLog}
+                    data-checked-state={isShowFullLog}
+                    onChange={(checked: boolean) => {
+                      toggleShowFullLog(checked);
                     }}
                   />
-                </React.Fragment>
-              );
-            })}
-          <Checkbox
-            label={t('public~Wrap lines')}
-            id="wrapLogLines"
-            isChecked={isWrapLines}
-            data-checked-state={isWrapLines}
-            onChange={(checked: boolean) => {
-              toggleWrapLines(checked);
-            }}
-          />
-          <Divider
-            orientation={{
-              default: 'vertical',
-            }}
-          />
-          <a href={currentLogURL} target="_blank" rel="noopener noreferrer">
-            <OutlinedWindowRestoreIcon className="co-icon-space-r" />
-            {t('public~Raw')}
-          </a>
-          <Divider
-            orientation={{
-              default: 'vertical',
-            }}
-          />
-          <a href={currentLogURL} download={`${resource.metadata.name}-${containerName}.log`}>
-            <DownloadIcon className="co-icon-space-r" />
-            {t('public~Download')}
-          </a>
-          {screenfull.enabled && (
-            <>
-              <Divider
-                orientation={{
-                  default: 'vertical',
-                }}
-              />
-              <Button variant="link" isInline onClick={toggleFullscreen}>
-                {isFullscreen ? (
-                  <>
-                    <CompressIcon className="co-icon-space-r" />
-                    {t('public~Collapse')}
-                  </>
-                ) : (
+                </Tooltip>
+              </DropdownItem>
+              <DropdownItem key={2} onClick={(ev) => ev.preventDefault()}>
+                <Checkbox
+                  label={t('public~Wrap lines')}
+                  id="wrapLogLines"
+                  isChecked={isWrapLines}
+                  data-checked-state={isWrapLines}
+                  onChange={(checked: boolean) => {
+                    toggleWrapLines(checked);
+                  }}
+                />
+              </DropdownItem>
+              <DropdownItem key={3}>
+                <a href={currentLogURL} target="_blank" rel="noopener noreferrer">
+                  <OutlinedWindowRestoreIcon className="co-icon-space-r" />
+                  {t('public~Raw')}
+                </a>
+              </DropdownItem>
+              <DropdownItem key={4}>
+                <a href={currentLogURL} download={`${resource.metadata.name}-${containerName}.log`}>
+                  <DownloadIcon className="co-icon-space-r" />
+                  {t('public~Download')}
+                </a>
+              </DropdownItem>
+              <DropdownItem key={5} onClick={(ev) => ev.preventDefault()}>
```

Comment:
```suggestion
              <DropdownItem key={toggle-action} onClick={(ev) => ev.preventDefault()}>
```

Reply:
> 

@jhadvig Hmm, not sure what you meant here! Where is `toggle-action` coming from?